packages/cli/test/mocks/secrets.ts

```diff
@@ -0,0 +1,29 @@
+import chance from 'chance';
+import { client } from './client';
+
+export function useSecrets({
```

Comment:
I don't love our current mock data/request setup but I did want to follow our existing pattern.